### DISCUSSION TITLE
Not sure if I'm doing wrong or bug

### DISCUSSION THREAD

**Discussion by CyberDrenDor** [xbapps/xbvr#1610] [General]
Hi everyone,

I wanted to check if this is just me not understanding expected behaviour or a bug before submitting a bug report for this.

I am trying to filter search for all scripts I have that don't have videos but it isn't working.

I have "Is Scripted" set to green and both "Single Video File" and "Multiple Video Files" set to red and search result is nothing, instead of all the scripts I have with no video files as I was expecting. I think my logic works as the results I want have no single or multiple video files but do have a script, yet nada.

Am I wrong in thinking this search should work or is this a glitch in XBVR that is not giving the search results it should? I am running the latest version of XBVR, version .4.21, just updated earlier today I believe.

Is there a different way of getting what I am looking for that I am perhaps overlooking or do you agree this is a bug report I should submit?

**Comment by theRealKLH** [comment]
remove all filters. click the "Scripted" button. Then click on "not downloaded". You should get all scenes (without vids) that you matched scripts to. ALSO... you get faster help on Discord. I just happened to be passing by...

**Reply by CyberDrenDor** [reply]
I had this all typed out and ready to send when I saw your reply:

Update, on second look I figured it out... I had to change my grouping on top from "Downloaded" to "Not Downloaded" and suddenly there are all my lonely scripts... guess the downloaded only applies to videos, not anything as I was first thinking. Apparently downloaded scripts don't count as downloaded...

Thanks for the reply though, and I did not know there was a Discord for this... but now that I know I will add it of course!
